### ISSUE TITLE
ANTLR Console should not kick in

### ISSUE THREAD

**Issue by rslemos** [jknack/antlr4ide#118]
When a grammar file is saved, antlr4ide starts compiling the grammar on background. That is nice.

What is not nice though is that the ANTLR Console kicks in, even when the view is minimized. Other consoles have buttons to control this behavior: "Show Console When Standard Out Changes" and "Show Console When Standard Error Changes".

**Comment by RedTailedHawk** [comment]
I agree.  I find myself entering input into a separate editor and then copying and pasting into the antlr4ide editor when I'm done.

Would be nice to get this fixed.  Thanks.

**Comment by dev0173** [comment]
For me, this behaviour is more of a problem when trying to test the grammar using the "Parse Tree" tab. Typing a test string normally result in the console grabbing focus before I've finished. I normally have to resort to preparing quick test strings externally then paste them in. Being able to turn off the console's "activate on write" feature would be great.

**Comment by StefanieScherzinger** [comment]
You can switch off the automatic build:

"Code is automatically generated on save if the grammar is valid. You can turn off this feature by going to: Window > Preferences > ANTLR 4 > Tool and uncheck the "Tool is activated" option. From there you can configure a couple more of options."

found here: https://github.com/jknack/antlr4ide

**Comment by rslemos** [comment]
@StefanieScherzinger I still think that automatically compiling the grammar is nice and really useful. What I miss is some control on whether the console will grab focus or not (see, other consoles have such controls). So I can have simultaneously both "automatic build" AND no console focus grabbing (only this last one is a nuisance).

**Comment by HSorensen** [comment]
antlr4ide should adhere to standard usage of console where the focus on changes to either stdout and stderr is optional.

**Comment by vbachvarov** [comment]
1+ for this bug.
The plugin is extremely useful and kuddos to the author. But the constant popping of the ANTLR4 console makes other views constantly to lose focus or get hidden behind it.
I really miss the two buttons of the normal console - activate when stdout changes, and activate when stderr changes. This should be configurable.
Another solution would be to just reuse the standard console and not have a special ANTLR4 one. This would automatically solve the bug.
